behavior.go

```diff
@@ -36,8 +36,10 @@ type ConsensusBehavior struct {
 		ProposalFreq       float64 `ask:"--proposal" help:"How often the engine gets to propose a block"`
 		FailedProposalFreq float64 `ask:"--ignore" help:"How often the payload produced by the engine does not become canonical"`
 		Finality           float64 `ask:"--finality" help:"How often an epoch succeeds to finalize"`
+		ReorgFrequency     float64 `ask:"--reorg-frequency" help:"Frequency of chain reorgs"`
```

Comment:
`--freq.reorg-frequency` is a bit redundant, maybe just change it to `--freq.reorg` ?

Reply:
oh, doh yeah it is